go/libraries/doltcore/env/actions/clone.go

```diff
@@ -264,7 +264,17 @@ func CloneRemote(ctx context.Context, srcDB *doltdb.DoltDB, remoteName, branch s
 		return err
 	}
 
-	ws := doltdb.EmptyWorkingSet(wsRef)
+	// Retrieve working set from srcDB
+	ws, err := srcDB.ResolveWorkingSet(ctx, wsRef)
```

Comment:
Definitely don't want to do this -- this will pull in uncommitted changes in the case of a local file system remote